x/concentrated-liquidity/model/msgs_test.go

```diff
@@ -26,42 +26,67 @@ func TestMsgCreateConcentratedPool(t *testing.T) {
 		{
 			name: "proper msg",
 			msg: clmodel.MsgCreateConcentratedPool{
-				Sender: addr1,
-				Denom0: "eth",
-				Denom1: "usdc",
+				Sender:      addr1,
+				Denom0:      "eth",
+				Denom1:      "usdc",
+				TickSpacing: 1,
 			},
 			expectPass: true,
 		},
 		{
 			name: "invalid sender",
 			msg: clmodel.MsgCreateConcentratedPool{
-				Sender: invalidAddr.String(),
-				Denom0: "eth",
-				Denom1: "usdc",
+				Sender:      invalidAddr.String(),
+				Denom0:      "eth",
+				Denom1:      "usdc",
+				TickSpacing: 1,
 			},
 			expectPass: false,
 		},
 		{
 			name: "missing denom1",
 			msg: clmodel.MsgCreateConcentratedPool{
-				Sender: invalidAddr.String(),
-				Denom0: "eth",
+				Sender:      invalidAddr.String(),
+				Denom0:      "eth",
+				TickSpacing: 1,
 			},
 			expectPass: false,
 		},
 		{
 			name: "missing denom0",
 			msg: clmodel.MsgCreateConcentratedPool{
-				Sender: invalidAddr.String(),
-				Denom1: "usdc",
+				Sender:      invalidAddr.String(),
+				Denom1:      "usdc",
+				TickSpacing: 1,
 			},
 			expectPass: false,
 		},
 		{
 			name: "missing sender",
 			msg: clmodel.MsgCreateConcentratedPool{
-				Denom0: "eth",
-				Denom1: "usdc",
+				Denom0:      "eth",
+				Denom1:      "usdc",
+				TickSpacing: 1,
+			},
+			expectPass: false,
+		},
+		{
+			name: "tick spacing less than minimum",
+			msg: clmodel.MsgCreateConcentratedPool{
+				Sender:      addr1,
+				Denom0:      "eth",
+				Denom1:      "usdc",
+				TickSpacing: 0,
```

Comment:
Let's define these minimums and maximums as constants in the production code and import them here please. So that if they get updated, we don't need to update the tests.

Also, where is the current choice of min and max documented? How was it chosen?